vllm/config.py

```diff
@@ -876,8 +876,8 @@ def __init__(
             from vllm.executor import ray_utils
             backend = "mp"
             ray_found = ray_utils.ray_is_available()
-            if (torch.cuda.is_available()
-                    and cuda_device_count_stateless() < self.world_size):
```

Comment:
we can use `if current_platform.is_cuda()`

Reply:
`cuda_device_count_stateless()` can raise error in other platforms

Reply:
Thanks, this is definitely better to use and seems to work locally